sdk/tools/spec2def/spec2def.c

```diff
@@ -711,52 +744,151 @@ OutputLine_def(FILE *fileDest, EXPORT *pexp)
     return 1;
 }
 
-int
-ParseFile(char* pcStart, FILE *fileDest, PFNOUTLINE OutputLine)
+void
+Fatalv(
+    const char* filename,
+    unsigned nLine,
+    const char *pcLine,
+    const char *pc,
+    unsigned errorlen,
+    const char *format,
+    va_list argptr)
 {
-    char *pc, *pcLine;
-    int nLine;
+    unsigned i, errorpos, len;
+
+    /* Get the length of the line */
+    for (len = 0; pcLine[len] != '\x0D' && pcLine[len] != '\x0A'; len++)
```

Comment:
Consider `strpbrk` (or, I suppose, `NextLine`)

Reply:
NextLine might be 1 or 2 chars after the end of the line. But strpbrk is nice.